templates/TEMPLATE_terraform.tfvars

```diff
@@ -606,14 +606,16 @@ flag_disable_email_login               = false
 
 /*
 ## ------------------------------------------------------------------------------------
-## EC2 - Docker Driver Logging
+## EC2 - Docker Configuration
 ## ------------------------------------------------------------------------------------
 # https://docs.docker.com/config/containers/logging/local/
 */
 flag_docker_logging_local              = false
 flag_docker_logging_journald           = true
 flag_docker_logging_jsonfile           = false
 
```

Comment:
Comment to document purpose of new parameter and when to use/not use/implications etc.